### DISCUSSION TITLE
v0.29.0-rc0

### DISCUSSION THREAD

**Discussion by borg323** [LeelaChessZero/lc0#1722] [General]
In this release:
* Initial support for attention policy, only cuda backend and partially in
  blas/dnnl/eigen (good enough for T79).
* Non multigather (legacy) search code and `--multigather` option are removed.
* 15b default net is now 753723.
* The onnx backend now allows selecting gpu to use.
* Improved error messages for unsupported network files.
* Some assorted fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/LeelaChessZero/lc0/releases/tag/v0.29.0-rc0'>v0.29.0-rc0</a>.</em>

**Comment by Vukica14** [comment]
Hi to all which version I should use for Nvidia GTX850M ? CUDAN ?

**Reply by mooskagh** [reply]
As per https://lczero.org/play/download/, it should be a cudnn version.
